### ISSUE TITLE
Proposal Creation - Name of the options is not good

### ISSUE THREAD

**Issue by Filipv95** [DXgovernance/DAVI#117]
**Description**
GIven I have created Contributor proposal and added two options ('Yes' and 'No') which have Transfer&Mint action, when I go to Proposal Preview page, options name, in Description part, is not one I have given.

**Steps to Reproduce**
1. SpinUp Localhost env
2. Open localhost:3000
3. Check if Landing Page is loaded
4. Click on SWPRGuild box
5. Check if SWPRGuild page is opened
6. Connect Wallet
7. Click on Create Proposal button
8. Create Contributor Propsal with two options 
9. Add Transfer&Mint actions to both Options
10. Finish creating Proposal and approve transaction on Metamast
11. Go to newly created Proposal Page 

**Expected behavior**
Name of the Options should be the one I have entere while creating the Proposal

**Actual behavior**
Wrong name of the proposal i.e "Against"

**Screenshots**
![dao1](https://user-images.githubusercontent.com/37701286/177836001-6ecbb6aa-8b07-43eb-96ea-f13389e60fdd.png)

**Device Information**
OS: MacOs
Browser GoogleChrome, Brave
Device: MacOS Monterey v12.4

**Additionaly context**
Basically I have tried same flow several times and this bug ocurrs from time to time.

**Comment by rossneilson** [comment]
Same issue as #65

**Comment by Filipv95** [comment]
Ack, thanks Ross, missed that.
